packages/@aws-cdk/aws-apigatewayv2-authorizers/lib/http/jwt.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  HttpAuthorizer,
+  HttpAuthorizerType,
+  HttpRouteAuthorizerBindOptions,
+  HttpRouteAuthorizerConfig,
+  IHttpAuthorizer,
+  IHttpRouteAuthorizer,
+} from '@aws-cdk/aws-apigatewayv2';
+import { Resource, Token } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+
+/**
+ * Properties to initialize HttpJwtAuthorizer.
+ */
+export interface HttpJwtAuthorizerProps {
+
+  /**
+   * The name of the authorizer
+   * @default 'JwtAuthorizer'
+   */
+  readonly authorizerName?: string;
+
+  /**
+   * The identity source for which authorization is requested.
+   *
+   * @default ['$request.header.Authorization']
+   */
+  readonly identitySource?: string[],
+
+  /**
+   * A list of the intended recipients of the JWT.
+   * A valid JWT must provide an aud that matches at least one entry in this list.
+   */
+  readonly jwtAudience: string[]
+
+  /**
+   * The base domain of the identity provider that issues JWT.
+   */
+  readonly jwtIssuer: string;
+}
+
+/**
+ * Authorize Http Api routes on whether the requester is registered as part of
+ * an AWS Cognito user pool.
+ */
+export class HttpJwtAuthorizer implements IHttpRouteAuthorizer {
+  /**
+   * Import an existing HTTP Jwt Authorizer into this CDK app.
+   */
+  public static fromHttpJwtAuthorizerId(scope: Construct, id: string, authorizerId: string): IHttpRouteAuthorizer {
```

Comment:
This method isn't required. 

You can just update the `fromHttpAuthorizerAttributes()` API in `HttpAuthorizer` class to implement `IHttpRouteAuthorizer`.

Reply:
So let me understand this. This means when creating a new authorizer we use `new HttpJwtAuthorizer()` but when importing an existing we use `HttpAuthorizer.fromHttpAuthorizerAttributes()` ?

Reply:
https://github.com/aws/aws-cdk/pull/10972/commits/ebae46583bfb38d2ddeb492d5dded436f3e098ca

Reply:
Sorry for the late reply, but yes. This is the pattern for all CDK constructs.